support-frontend/assets/helpers/abTests/abtest.ts

```diff
@@ -320,15 +337,21 @@ function getParticipations(
 				mvtId,
 				country,
 				countryGroupId,
-				acquisitionDataTest,
+				acquisitionDataTests,
 			)
 		) {
 			const variantIndex = assignUserToVariant(
 				mvtId,
 				test,
-				acquisitionDataTest,
+				testId,
+				acquisitionDataTests,
 			);
-			participations[testId] = test.variants[variantIndex].id;
+
+			if (variantIndex === -1) {
+				participations[testId] = notintest;
+			} else {
+				participations[testId] = test.variants[variantIndex].id;
+			}
```

Comment:
Not checking if the `variantIndex` is `-1` actually breaks the page due to the `.id` call. We've now wrapped the assignment in this check